ctpolicy/applepolicy.go

```diff
@@ -40,14 +39,16 @@ func (appleP AppleCTPolicy) LogsByGroup(cert *x509.Certificate, approved *loglis
 		incCount = 5
 	}
 	baseGroup, err := BaseGroupFor(approved, incCount)
+	if err != nil {
+		return nil, err
+	}
 	groups := LogPolicyData{baseGroup.Name: baseGroup}
 	return groups, err
```

Comment:
I think this can explicitly be `groups, nil` now, just for clarity.
Same elsewhere too.

Reply:
Done